apps/web/pages/[chain].tsx

```diff
@@ -17,7 +17,8 @@ interface Props {
 const Home: NextPage<Props> = ({ isOutOfCELO, network }: Props) => {
   const networkCapitalized = capitalize(network)
 
-  const otherNetwork = network === 'alfajores' ? 'cannoli' : 'alfajores'
+  const otherNetwork =
+    networks.indexOf(network) === 0 ? networks[1] : networks[0]
```

Comment:
Nit: Haven't run this locally to check, but does `networks[1]` work where networks only has one element?

```ts
export const networks = ['alfajores']
```

Reply:
when there is only 1 element other network would be defined as undefined. however down below i also dont render the switch network button if there is only one network in the array.